### DISCUSSION TITLE
git source control for IRIS

### DISCUSSION THREAD

**Discussion by Bitnerd79** [intersystems/ipm#431] [Q&A]
I am trying to install git-source-control on an InterSystems IRIS server. I got zpm/ipm installed by downloading the file and importing it into IRIS. But the zpm command seems to require access to pm.community.intersystems.com and my server does not allow that. Is there a way to download the necessary files and install them so I can use git-source-control?

**Comment by isc-tleavitt** [comment]
If you clone git-source-control - either by having access to github.com and cloning on your IRIS server, or by cloning locally than transferring the files using your approach of choice - you can install by running:
`zpm "load /path/to/package/folder"`
where /path/to/package/folder contains module.xml (and all the other files from the Git repo)

**Reply by isc-tleavitt** [reply]
(note, this would be generally true for any IPM package, so this is just as good a place for the discussion as git-source-control's discussions area itself)

**Comment by Bitnerd79** [comment]
Thank you.

From: Tim Leavitt ***@***.***>
Sent: Monday, October 23, 2023 1:48 PM
To: intersystems/ipm ***@***.***>
Cc: Alan Wayne Watts ***@***.***>; Author ***@***.***>
Subject: [External Email] Re: [intersystems/ipm] git source control for IRIS (Discussion #431)

CAUTION: This message originated from outside of MediQuant. Do not click on any links or open attachments unless you were expecting an email from the sender and know it is safe.


(note, this would be generally true for any IPM package, so this is just as good a place for the discussion as git-source-control's discussions area itself)

-
